operators/bosh-operator/DirectorService.js

```diff
@@ -953,6 +953,8 @@ class DirectorService extends BaseDirectorService {
   }
 
   configureAddOnJobs(serviceManifest, context) {
+    _.set(context, 'serviceId', this.service.id);
+    _.set(context, 'connectivityAllowedServicesList', ['6db542eb-8187-4afc-8a85-e08b4a3cc24e']);
```

Comment:
Can we remove this hardcoded part then ?

Reply:
sure, I will modify to accept this as part of service catalog attribute, and will enable for postgresql for now.